docs/types/response.md

```diff
@@ -3,42 +3,110 @@
 
 @signature `response(status, body, headers, statusText)`
 
-Used to detail a response.
+  Used to detail a response.
 
-Example:
+  Example:
```

Comment:
We should always introduce what the example is doing unless it is completely obvious.

Reply:
Resolved in 162c8fa.